100of100Service/src/main/java/dev/coms4156/project/command/AddEmployeeToDeptCommand.java

```diff
@@ -0,0 +1,63 @@
+package dev.coms4156.project.command;
+
+import dev.coms4156.project.Department;
+import dev.coms4156.project.Employee;
+import dev.coms4156.project.HrDatabaseFacade;
+import dev.coms4156.project.exception.NotFoundException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * A command to add an employee to given department.
+ */
+public class AddEmployeeToDeptCommand implements Command {
+  private final int clientId;
+  private final int departmentId;
+  private final String name;
+  private final String hireDate;
+
+  /**
+   * Constructs a command to add an employee to a given department.
+   *
```

Comment:
This is a great and powerful endpoint, but I may think the `AddEmployeeToDept` will be used to assign an **existing employee** to an **existing department**. And maybe another endpoint is called `CreateEmployee` to create an employee but only add it to the organization without assigning it to a department.